docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+
+  backend:
+    image: python:3.5
+    command: python3 /app/refactor_me.py
+    environment:
+      - PYTHONUNBUFFERED=1
+    volumes:
+      - ./:/app
+
+  flake:
+    image: flake:anton
```

Comment:
is this image publicly available? how other developers supposed to use it?

Reply:
this is for my local usage, but if you think it may be usefull , we can share it on docker hub

Reply:
but first, we need to refactor, to make it convenient for you at least, bcz i have it working in my local not standard build :) Most programmers are wokring on MacOS, using MacBooks Pro, I'm using PC + Intel NUC. PC for editor and browser, NUC with Ubuntu inside, Docker and share via samba...